MapboxSearch/offline/src/main/java/com/mapbox/search/offline/OfflineSearchEngine.kt

```diff
@@ -205,6 +205,46 @@ public interface OfflineSearchEngine {
         callback = callback,
     )
 
+    /**
+     * Performs a search along the supplied [route].
```

Comment:
nit

If this implementation is [experimental](https://github.com/mapbox/mapbox-search-android/pull/201/files#r1529013017) it makes sense to mention this in doc.